dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml

```diff
@@ -0,0 +1,32 @@
+%h3 COVID-19 precautions
+
+%p
+  In light of the global situation surrounding the COVID-19 virus, we understand that traveling and
+  gathering for a workshop may be cause for anxiety or concern for teachers registered for upcoming
+  Code.org workshops. We encourage you to refer to the
+  = link_to 'Centers for Disease Control and Prevention', 'https://www.cdc.gov/', target: '_blank'
+  (CDC) for the most up to date recommendations, as is your Regional Partner in evaluating whether
+  to host your workshops as planned. If you have returned from a
```

Comment:
"as is" doesn't read right.  Suggest something like:

We encourage you to refer to the Centers for Disease Control and Prevent (CDC) for the most up-to-date recommendations.  Your regional partner will do the same when evaluating whether to host your workshops as planned.